script/MerkleRootCreation/Bob/CreateHybridBtcMerkleRoot.s.sol

```diff
@@ -35,18 +35,18 @@ contract CreateHybridBtcMerkleRoot is Script, MerkleTreeHelper {
         setAddress(false, bob, "accountantAddress", accountantAddress);
         setAddress(false, bob, "rawDataDecoderAndSanitizer", rawDataDecoderAndSanitizer);
 
-        ManageLeaf[] memory leafs = new ManageLeaf[](16);
+        ManageLeaf[] memory leafs = new ManageLeaf[](32);
 
 
         // ========================== Standard Bridge ==========================
         
-         ERC20[] memory localTokens = new ERC20[](2);
+        ERC20[] memory localTokens = new ERC20[](2);
         localTokens[0] = getERC20(sourceChain, "WBTC");
         localTokens[1] = getERC20(sourceChain, "LBTC");
 
         ERC20[] memory remoteTokens = new ERC20[](2);
-        remoteTokens[0] = getERC20(bob, "WBTC");
-        remoteTokens[1] = getERC20(bob, "LBTC");
+        remoteTokens[0] = getERC20(mainnet, "WBTC");
```

Comment:
Was this the issue with the bridge deployment? The remote chain should have been Ethereum since the receipt token is minted on BOB.